LanternExtractor/EQ/Wld/Exporters/GltfWriter.cs

```diff
@@ -966,10 +967,11 @@ private IMeshBuilder<MaterialBuilder> InstantiateMeshBuilder(string meshName, bo
             var vertex0 = GetGltfVertex<TvG, TvM, TvS>(vertexPositions.v0, vertexNormals.v0, vertexUvs.v0, vertexColors.v0, isSkinned, boneIndexes.v0);
             var vertex1 = GetGltfVertex<TvG, TvM, TvS>(vertexPositions.v1, vertexNormals.v1, vertexUvs.v1, vertexColors.v1, isSkinned, boneIndexes.v1);
             var vertex2 = GetGltfVertex<TvG, TvM, TvS>(vertexPositions.v2, vertexNormals.v2, vertexUvs.v2, vertexColors.v2, isSkinned, boneIndexes.v2);
-            if (isSkinned)
+
+            // Always use clockwise rotation to offset the mirrored x axis
+            // If we're embedding in a zone or applying to a skinned model
+            if (objectInstance != null || isSkinned || isZoneMesh)
```

Comment:
So at this point, the only time we don't mirror the X axis and reverse the triangle winding order is when the exporter writes stand-alone non-skinned objects. Outside the scope of this PR, but I'm thinking this should probably apply universally.

Reply:
Yes it seems like those are animated objects from what I could tell in testing. I wonder why this is, i.e. is there something else occurring during translating and building the meshes in that case that requires this that could be taken care of upstream?